error-guides/parser/invalid_package_name.md

```diff
@@ -0,0 +1,28 @@
+# Invalid package name
+
+## Example
+
+This error occurs when a package name in an `import` contains invalid characters.
+
+Erroneous code example:
+
+```js
+import FOO.bar;
```

Comment:
@gluax After merging #1623 is this error message now obsolete?

Reply:
Correct the parser no longer yells for capital idents.

Reply:
@Centril Can you please update the PR to reflect this?

Reply:
Oh, I guess we can just delete this file then. Will do.